app/styles/modules/_settings-account-recovery.scss

```diff
@@ -0,0 +1,165 @@
+.account-recovery-panel {
+  .list {
+    ul {
+      padding: 0;
+    }
+
+    li {
+      list-style: none;
+    }
+
+    .type {
+      height: 40px;
+      margin: 10px 0;
+      position: relative;
+
+      html[dir='ltr'] & {
+        background-position: left 2px;
+      }
+
+      html[dir='rtl'] & {
+        background-position: right 2px;
+      }
+
+      .name {
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+
+        @include respond-to('big') {
+          width: calc(95% - 95px);
+        }
+
+        @include respond-to('small') {
+          width: 100%;
+        }
+      }
+
+      .details {
+        color: $color-grey;
+        font-size: 12px;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        width: calc(95% - 95px);
+
+        & .disabled {
+          color: $color-grey;
+        }
+
+        & .enabled {
+          color: $color-green;
+        }
+      }
+
+      .settings-button-group {
```

Comment:
The `.details` and `.settings-button-group` blocks are duplicated 4x in the scss now, they should be extracted to reduce duplication.

Reply:
I combined the `.details` for emails, account recovery and totp views. However, for `.settings-button-group` I can only combine account recovery and totp since they are similar panels in terms of button layout. The emails panel has multiple buttons on the row and would have caused the scss to be kinda wonky.